### DISCUSSION TITLE
Any reason to keep the AAX files?

### DISCUSSION THREAD

**Discussion by wdpcpa** [openaudible/openaudible#724] [Q&A] [answered]
I see OpenAudible downloads the Audible file and then converts it to an MP3.  Is there any reason to keep the AAX file after the conversion is done?  Or should I just delete them?

**Comment by openaudible** [comment, marked as answer]
I like to keep them. 
You may decide you want to re-encode the files (for instance, because we recently added support for chapter names in the meta data) its good to have them.
You might also decide you like m4b files instead of mp3 (better quality, faster conversions.)

But you can delete them and OA won't download them unless you delete the converted version.

**Reply by wdpcpa** [reply]
Thanks.  I will have to understand M4b files better.  I no longer use Itunes for playback.  I use Smart Audio Player.  I don't know how they would work with the app.  Nice thing about MP3 is they seem to work with anything.  

I have space for the AAX files right now but will probably delete them from my backup.  Audible can be my backup.  My NAS is getting pretty full.  Nice to know that it won't mess up the sync if I dump them.  That was a concern.

**Comment by oboe45** [comment]
I prefer to retain the local copy of AAX files - downloading over 1000 AAX
files from Audible takes the best part of two days, so a local AAX disk is
much more practical.  Even with a very very fast download and multiple
download threads, Audible limits download speed to a very pedestrian rate.

On Fri, Sep 24, 2021 at 12:18 PM wdpcpa ***@***.***> wrote:

> Thanks. I will have to understand M4b files better. I no longer use Itunes
> for playback. I use Smart Audio Player. I don't know how they would work
> with the app. Nice thing about MP3 is they seem to work with anything.
>
> I have space for the AAX files right now but will probably delete them
> from my backup. Audible can be my backup. My NAS is getting pretty full.
> Nice to know that it won't mess up the sync if I dump them. That was a
> concern.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openaudible/openaudible/discussions/724#discussioncomment-1380937>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APZFLTJVF47T42BKPXWARCDUDSQFRANCNFSM5EQHPA4A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
-- 
Bob Barker

**Reply by wdpcpa** [reply]
Thanks Bob.  That's 2 votes for keeping the AAX files.  Do you backup the AAX files as well?

**Comment by oboe45** [comment]
Yes, I had a lot of trouble initially with OpenAudible usage - mostly all
my own ignorance - so I have multiple AAX copies on local NAS’s and extrav
standalone disks - cheaper and faster than repeating the download (which I
have done at least twice!). M4B conversion is very fast, so I do not retain
extra M4b file backups -  MP3 conversion is abysmally slow so I do not
convert to Mp3 and I only have a few legacy Mp3 files from way back.

On Fri, Sep 24, 2021 at 12:29 PM wdpcpa ***@***.***> wrote:

> Thanks Bob. That's 2 votes for keeping the AAX files. Do you backup the
> AAX files as well?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/openaudible/openaudible/discussions/724#discussioncomment-1380998>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APZFLTLON2YOMYYQ25P56U3UDSRN5ANCNFSM5EQHPA4A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
-- 
Bob Barker
